utils/memusage.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-env es6, node */
+/* globals gc */
+
+'use strict';
+
+if (typeof gc !== 'function') {
+	throw new Error('Please re-run with `node --expose-gc utils/memusage.js`');
+}
+
+const { RuleSets } = require('../chromium/rules');
```

Comment:
@Hainish @bcyphers Did this script work at the point of merging? This require seems outdated as `rules.js` now lives in `background-scripts` folder.